server/src/main/java/org/elasticsearch/cluster/coordination/ElectionSchedulerFactory.java

```diff
@@ -180,7 +181,15 @@ public String toString() {
             };
 
             logger.debug("scheduling {}", runnable);
-            threadPool.schedule(TimeValue.timeValueMillis(delayMillis), Names.GENERIC, runnable);
+            try {
+                threadPool.schedule(TimeValue.timeValueMillis(delayMillis), Names.GENERIC, runnable);
+            } catch (EsRejectedExecutionException e) {
```

Comment:
This change is just for `EsRejectedExecutionException`-handling, which arose in test failures. No test to verify this specifically is added here.